pip/flatpak-pip-generator

```diff
@@ -129,6 +130,15 @@ def parse_continuation_lines(fin):
         yield line
 
 
+def get_pypi_checker_data(name: str, url: str) -> Dict[str, str]:
+    checker_data = {"type": "pypi", "name": name}
+    if url.endswith(".whl"):
+        checker_data["packagetype"] = "bdist_wheel"
+    elif url.endswith(".tar.gz"):
```

Comment:
This does include all source file extension, you probably should check all of them, or just use `else:` here.